src/react-native/react-native-wizard.ts

```diff
@@ -210,11 +210,27 @@ Or setup using ${chalk.cyan(
     );
   }
 
+  // Ask if user wants to enable Logs
+  const enableLogs = await abortIfCancelled(
```

Comment:
l: I think at some point we could switch to using `featureSelectionPrompt` like in most other wizards. But no need to do this right now!